packages/prosess-uttak/src/ui/UttakContainer.tsx

```diff
@@ -44,43 +45,48 @@ const UttakContainer = ({ containerData }: MainComponentProps): JSX.Element => {
 
   return (
     <ContainerContext.Provider value={containerData}>
-      <HStack justify="start" className={styles.overstyringsHeader}>
-        <Heading size="small" level="1">
-          Uttak
-        </Heading>
-        {erOverstyrer && <OverstyringKnapp erOverstyrt={overstyringAktiv} onClick={toggleOverstyring} />}
-      </HStack>
+      <VStack gap="4">
+        <HStack justify="start" className={styles.overstyringsHeader}>
+          <Heading size="small" level="1">
+            Uttak
+          </Heading>
+          {erOverstyrer && <OverstyringKnapp erOverstyrt={overstyringAktiv} onClick={toggleOverstyring} />}
+        </HStack>
+        <Infostripe harVentAnnenPSBSakAksjonspunkt={harVentAnnenPSBSakAksjonspunkt} />
 
-      <Infostripe harVentAnnenPSBSakAksjonspunkt={harVentAnnenPSBSakAksjonspunkt} />
+        {vurderOverlappendeSakComponent && (
+          <div className={styles.overlappendeSakContainer}>{vurderOverlappendeSakComponent}</div>
+        )}
 
-      <OverstyrUttakContextProvider>
-        <OverstyrUttakForm overstyringAktiv={overstyringAktiv} />
-      </OverstyrUttakContextProvider>
+        <OverstyrUttakContextProvider>
+          <OverstyrUttakForm overstyringAktiv={overstyringAktiv} />
+        </OverstyrUttakContextProvider>
 
-      <UtsattePerioderStripe />
-      {/* Allerede løst og har klikket rediger, eller har uløst aksjonspunkt */}
-      {((virkningsdatoUttakNyeRegler && redigerVirkningsdato) ||
-        harAksjonspunktVurderDatoMedStatusOpprettet ||
-        (readOnly && aksjonspunktVurderDato)) && (
-        <VurderDato
-          avbryt={
-            virkningsdatoUttakNyeRegler && redigerVirkningsdato ? () => setRedigervirkningsdato(false) : undefined
-          }
-          initialValues={{
-            begrunnelse: aksjonspunktVurderDato?.begrunnelse,
-            virkningsdato: virkningsdatoUttakNyeRegler,
-          }}
-          readOnly={readOnly}
-        />
-      )}
-      {!harVentAnnenPSBSakAksjonspunkt && (
-        <UttaksperiodeListe
-          uttaksperioder={lagUttaksperiodeliste(uttaksperioder, inntektsgraderinger)}
-          redigerVirkningsdatoFunc={() => setRedigervirkningsdato(true)}
-          redigerVirkningsdato={redigerVirkningsdato}
-          readOnly={readOnly}
-        />
-      )}
+        <UtsattePerioderStripe />
+        {/* Allerede løst og har klikket rediger, eller har uløst aksjonspunkt */}
+        {((virkningsdatoUttakNyeRegler && redigerVirkningsdato) ||
+          harAksjonspunktVurderDatoMedStatusOpprettet ||
```

Comment:
Kan vurdere å trekke dette ut i en const så blir det mer lesbart her

Reply:
Forsåvidt, men ikke noe som er lagt til nå dette da 😄

Reply:
Jeg endrer ikke på dette i denne PR'en da det ikke er relevant for søskensaker